src/test/resources/fixtures/rest/users/user.json

```diff
@@ -94,5 +95,6 @@
   "default_profile_image": false,
   "following": false,
   "follow_request_sent": false,
-  "notifications": false
+  "notifications": false,
+  "translator_type": "regular"
```

Comment:
I've checked and we do not seem to support the field `translator_type` in user...we should! I would expect the test to detect missing supported fields...what am I missing?

Reply:
I ran out of time last evening to add the `profile_location` and `translator_type` fields (that are present in this one: https://developer.twitter.com/en/docs/accounts-and-users/follow-search-get-users/api-reference/get-users-show). Added now. 🙂